File: gluecodium/src/test/resources/smoke/visibility_attribute/output/android/com/example/smoke/InternalLambdaImpl.java
/*
 *
 */
package com.example.smoke;
import com.example.NativeBase;
class InternalLambdaImpl extends NativeBase implements InternalLambda {
    protected InternalLambdaImpl(final long nativeHandle, final Object dummy) {
        super(nativeHandle, new Disposer() {
            @Override
            public void disposeNative(long handle) {
                disposeNativeHandle(handle);
            }
        });
    }
    private static native void disposeNativeHandle(long nativeHandle);
    public native void apply();
}

File: gluecodium/src/test/resources/smoke/visibility_attribute/output/android-kotlin/com/example/smoke/InternalLambdaImpl.kt
/*

 *
 */

@file:JvmName("InternalLambdaImplExtensions")


package com.example.smoke

import com.example.NativeBase

/**
 * @suppress
 */
internal class InternalLambdaImpl : NativeBase, InternalLambda {
    protected constructor(nativeHandle: Long, @Suppress("UNUSED_PARAMETER") tag: Any?)
        : super(nativeHandle, { disposeNativeHandle(it) }) {}



    @JvmSynthetic override external fun apply() : Unit



    companion object {
        @JvmStatic private external fun disposeNativeHandle(nativeHandle: Long)
    }
}
